documents/TestSuite/stdlib/geometric/geomattrvalue.mtlx

```diff
@@ -0,0 +1,64 @@
+<!-- geomattrvalue unit test -->
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <output name="geomattrvalue_integer_out" type="vector4" nodename="switch1" xpos="12.4207" ypos="4.42" />
+  <switch name="switch1" type="vector4" xpos="5.74483" ypos="4.12">
+    <input name="in1" type="vector4" value="0.0, 0.0, 0.0, 0.0" />
+    <input name="in2" type="vector4" value="0.3000, 0.3000, 0.3000, 0.3000" />
+    <input name="in3" type="vector4" value="0.5000, 0.5000, 0.5000, 0.5000" />
+    <input name="in4" type="vector4" value="0.7000, 0.7000, 0.7000, 0.7000" />
+    <input name="in5" type="vector4" value="1.0000, 1.0000, 1.0000, 1.0000" />
+    <parameter name="which" type="integer" nodnname="geomattrvalue_integer" />
+  </switch>
+  <geomattrvalue name="geomattrvalue_integer" type="integer" xpos="9.00908" ypos="16.4387">
+    <parameter name="attrname" type="string" value="geomattrvalue_integer" />
+  </geomattrvalue>
+
+  <output name="geomattrvalue_boolean_out" type="color3" nodename="switch2" xpos="12.4886" ypos="11.0706" />
```

Comment:
We allow building of shaders which have bool, int, and string output which fails when trying to render. Think we should do a check on the final output if this is disallowed. Not sure as this is with testRender.